Description: build only non-gzipped versions of semver for browser
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2013-08-16
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,5 @@
 files =  semver.browser.js \
-         semver.min.js \
-				 semver.browser.js.gz \
-				 semver.min.js.gz
+         semver.min.js
 
 all: $(files)
 
@@ -16,7 +14,7 @@
 		cat foot.js ) > semver.browser.js
 
 semver.min.js: semver.browser.js
-	uglifyjs -m <semver.browser.js >semver.min.js
+	uglifyjs semver.browser.js >semver.min.js
 
 %.gz: %
 	gzip --stdout -9 <$< >$@
